docker-compose.test.yml

```
@@ -0,0 +1,45 @@
+---
+# Compose file for test runs
+version: "3.7"
+services:
+  db:
+    image: datarttu/sujuikodb:latest
+    build: .
+    restart: unless-stopped
+    env_file: .env_test
+    volumes:
+      - type: bind
+        source: ./db/db_test
+        target: /docker-entrypoint-initdb.d/
+        read_only: true
+      - type: tmpfs
+        target: /var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD", "pg_isready", "-q", "-U", "${POSTGRES_USER}"]
```

```suggestion
      test: ["CMD", "pg_isready", "-q"]
```
I found that the healthcheck was unable to find "${POSTGRES_USER}", gave a warning and continued anyway. Removing it removed the warning.